cairoplot/seriestests.py

```diff
@@ -1,7 +1,7 @@
 import cairo, math, sys
 
 import cairoplot
-from .series import Series
+.from series import Series
```

Comment:
Upstream doesn't have a `.` before `from`

Reply:
Yeah, thanks, I missed that.